src/main/java/me/mrCookieSlime/Slimefun/api/SlimefunItemStack.java

```diff
@@ -150,6 +151,15 @@ public String getItemID() {
         return id;
     }
 
+    /**
+     * Gets the {@link SlimefunItem} associated for this SlimefunItemStack. Null if no item is found.
+     *
+     * @return The {@link SlimefunItem} for this SlimefunItemStack, null if not found.
+     */
+    public SlimefunItem getItem() {
```

Comment:
I'd probably even name it "getSlimefunItem()".
Also why is this build failing?

Reply:
Very stupid things, fixed now. 

The reason I didn't name it that is, this is already an ItemStack and getting the ID is getItemID. Follows the pattern of using getItem. 

Personally, it should be both changed or keeping this like so.